book/src/content/docs/deployment/opamp.md

```
@@ -0,0 +1,143 @@
+cd opamp-go/internal/examples/server
+go run .
+# Server starts on 0.0.0.0:4320
+
+# Point ffwd at it
+ff run --config ffwd.yaml
+# (with opamp.endpoint: http://localhost:4320/v1/opamp in the config)
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)